pyproject.toml

```diff
@@ -77,7 +78,7 @@ packages = ["merlin", "merlin.core", "merlin.datasets", "merlin.pcvl_pytorch", "
 
 [tool.pytest.ini_options]
 testpaths = ["tests"]
-python_files = ["test_*.py"]
+python_files = ["test_*.py", "benchmark_*.py"]
```

Comment:
the test coverage PR has moved all benchmarks files to benchmarks/, we can add "benchmarks" to testpaths